test/adversary.test.mjs

```diff
@@ -56,6 +62,8 @@ describe('Testing with an adversary', () => {
   // const value = 10;
   // const value1 = 1000;
   const value2 = 5;
+  let rollbackCount = 0;
+  const minExpectedRollbacks = 6;
```

Comment:
In a new issue please make `minExpectedRollbacks` equal to the number of bad transaction types that exist by configuring the adversary test to run good transaction(s) for each bad transaction such that there is a rollbacked block for each bad transaction type. This ensures testing each type of bad transaction.

Reply:
understood, I will make the changes in next PR